knora/excel2xml.py

```diff
@@ -1764,17 +1793,26 @@ def excel2xml(datafile: str, shortcode: str, default_ontology: str) -> None:
                 "permissions": resource_permissions,
                 "id": resource_id
             }
+            if check_notna(row.get("ark")):
+                kwargs_resource["ark"] = row["ark"]
+            if check_notna(row.get("iri")):
+                kwargs_resource["iri"] = row["iri"]
+            if check_notna(row.get("created")):
+                kwargs_resource["creation_date"] = row["created"]
             if resource_restype not in ["Annotation", "Region", "LinkObj"]:
                 kwargs_resource["restype"] = resource_restype
-                if check_notna(row.get("ark")):
```

Comment:
Until now, ARK and IRI were not supported for `<annotation>` and so on (see previous comment). But i think we just forgot it in the past. It should be supported.